main.go

```diff
@@ -28,14 +28,15 @@ import (
 	"k8s.io/klog/v2"
 	"sigs.k8s.io/controller-runtime/pkg/manager/signals"
 
-	"github.com/castai/cluster-controller/actions"
-	"github.com/castai/cluster-controller/castai"
-	"github.com/castai/cluster-controller/config"
-	"github.com/castai/cluster-controller/csr"
 	"github.com/castai/cluster-controller/health"
-	"github.com/castai/cluster-controller/helm"
-	"github.com/castai/cluster-controller/version"
-	"github.com/castai/cluster-controller/waitext"
+	"github.com/castai/cluster-controller/internal/actions/csr"
+	"github.com/castai/cluster-controller/internal/castai"
+	config2 "github.com/castai/cluster-controller/internal/config"
```

Comment:
Why `config2`?

Reply:
good catch, let me fix it (IDE did refactor on moving packages)